recipes/hbreader/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "hbreader" %}
+{% set version = "0.9.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/hbreader-{{ version }}.tar.gz
+  sha256: d2c132f8ba6276d794c66224c3297cec25c8079d0a4cf019c061611e0a3b94fa
+  patches:
+    - tests.patch
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+  skip: true  # [win or py<37]
```

Comment:
If this project isn't building any c-extensions, can it be a `noarch: python` build?
https://conda-forge.org/docs/maintainer/knowledge_base/#noarch-python

Then you can drop the `skip` and then just change the python requirement under `host` and `run` to `python >=3.7`

Reply:
The motivation here was that the windows tests are failing, but it's still worth releasing macOS/linux build for now.